// error
//? error is inevitable

// const x = 2;
// x = 5;
// console.log("Hi");

// ? error handling

// const x = 2;
// try {
//   x = 5;
//   console.log("Hit");
// } catch (error) {
//   console.log(error.message);
// } finally {
//   console.log("Send email");
// }
// console.log("Hi");

//? example 2

// try {
//   let user = { name: Arun };
//   if (!user) {
//     throw new Error("User doesn't exist");
//   }
//   console.log("Hey man, Wassup");
// } catch (error) {
//   console.log(error.message);
// }
